changelog/8.8.0.yaml

```diff
@@ -0,0 +1,287 @@
+version: 8.8.0
+entries:
+    - kind: enhancement
+      summary: async download of elastic-agent upgrade artifacts during fleet managed upgrade
+      description: ""
+      component: agent
+      pr:
+        - https://github.com/elastic/elastic-agent/pull/2205
+      issue: []
+      timestamp: 1674840786
+      file:
+        name: 1674840786-async-download-upgrade-artifacts.yaml
+        checksum: 0aa7b1bb3810df922c917dc5015b16a5a4a6d965
+    - kind: bug-fix
+      summary: Fix system config in example standalone config file
+      description: ""
+      component: config
+      pr:
+        - https://github.com/elastic/elastic-agent/pull/2338
+      issue: []
+      timestamp: 1677711770
+      file:
+        name: 1677711770-fix-standalone-system-config.yaml
+        checksum: ab73fc45885268717bc74899eca5d14c150a614a
+    - kind: bug-fix
+      summary: Fix parsing of paths from container-paths.yml file
+      description: ""
+      component: agent
+      pr:
+        - https://github.com/elastic/elastic-agent/pull/2340
+      issue: []
+      timestamp: 1677780422
+      file:
+        name: 1677780422-fix-container-paths-config-parsing.yaml
+        checksum: 99d209f37ce7d6c57d4d1d3f67016f22696faf28
+    - kind: feature
+      summary: Agent actions signature validation
+      description: ""
+      component: ""
+      pr:
+        - "2348"
+      issue: []
+      timestamp: 1678136865
+      file:
+        name: 1678136865-Agent-actions-signature-validation.yaml
+        checksum: 372b3c9abff7d7723e032653d46234143ac647c6
```

Comment:
I think this can be removed according to [Craig's comment](https://github.com/elastic/ingest-docs/pull/219#discussion_r1198285571) `This was disabled in https://github.com/elastic/elastic-agent/commit/ce0542fd641ea3e13d29bb2f88ea0ec403d0eed0 without removing the changelog fragment.`